code/modules/mob_spawn/ghost_roles/mining_roles.dm

```diff
@@ -323,3 +323,23 @@
 /obj/item/clothing/mask/chameleon/gps/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/gps, "Encrypted Signal")
+
+/obj/effect/mob_spawn/ghost_role/human/plasmaman
+	name = "plasma filled pod"
+	desc = "A sleeper retrofitted to use plasma instead of oxygen."
+	prompt_name = "plasmaman survivor"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "psykerpod"
+	density = TRUE
+	mob_species = /datum/species/plasmaman
+	you_are_text = "You are a plasmaman, a sentient colony of fungi created by one of the plasma pits outside."
+	flavour_text = "Dispite not knowing your past or who you once were, you were able to keep information even through your rebirth \
```

Comment:
```suggestion
	flavour_text = "Despite not knowing your past or who you once were, you were able to keep information even through your rebirth \
```

Reply:
also, for the record, players who are converted into plasmamen via plasma rivers retain their memories/are still the same person

Reply:
> also, for the record, players who are converted into plasmamen via plasma rivers retain their memories/are still the same person
> 
> mechanically, anyway, idk what our lore on it is

Ah, alright! I will update the description accordingly!